+nosnoc/+discrete_problem/stewart.m

```diff
@@ -0,0 +1,771 @@
+classdef stewart < vdx.problems.Mpcc
+    properties
+        model
+        dcs
+        opts
+
+        populated
+    end
+
+    methods
+        function obj = stewart(dcs, opts)
+            obj = obj@vdx.problems.Mpcc();
+            obj.model = dcs.model;
+            obj.dcs = dcs;
+            obj.opts = opts;
+        end
+
+        function create_variables(obj)
+            dims = obj.dcs.dims;
+            dcs = obj.dcs;
+            model = obj.model;
+            opts = obj.opts;
+
+            obj.p.rho_h_p = {{'rho_h_p',1}, 1};
+            obj.p.rho_terminal_p = {{'rho_terminal_p',1}, 1};
```

Comment:
just a remark: when we add generic constraint relaxation, then the penality parameter should be defined still here and then fed to vdx, or implicitly defined by vdx, when one defines a constraints a specifics that it must be relaxed.
also in this case, the parameter might be (1/sigma).